bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Journal.java

```diff
@@ -1118,6 +1129,7 @@ journalFormatVersionToWrite, getBufferedChannelBuilder(),
                                 if (entry != null && (!syncData || entry.ackBeforeSync)) {
                                     toFlush.set(i, null);
                                     numEntriesToFlush--;
+                                    entry.setEnqueueCbThreadPooleQueueTime(MathUtils.nowInNano());
```

Comment:
It only records the ack without flush items, we should also record the flushed items when `syncData = true`

Reply:
Fixed,PTAL,thanks!